tests/test_parquet.py

```diff
@@ -1708,5 +1710,39 @@ def testWriteNumpyDictBad(self):
         pass
 
 
+@unittest.skipUnless(np is not None, "Cannot test compute_row_group_size without numpy.")
+@unittest.skipUnless(pa is not None, "Cannot test compute_row_group_size without pyarrow.")
+class ComputeRowGroupSizeTestCase(unittest.TestCase):
+    """Tests for compute_row_group_size."""
+
+    def testRowGroupSizeNoMetadata(self):
+        numpyTable = _makeSimpleNumpyTable(include_multidim=True)
+
+        # We can't use the numpy_to_arrow convenience function because
+        # that adds metadata.
+        type_list = _numpy_dtype_to_arrow_types(numpyTable.dtype)
+        schema = pa.schema(type_list)
+        arrays = _numpy_style_arrays_to_arrow_arrays(
+            numpyTable.dtype,
+            len(numpyTable),
+            numpyTable,
+            schema,
+        )
+        arrowTable = pa.Table.from_arrays(arrays, schema=schema)
+
+        row_group_size = compute_row_group_size(arrowTable.schema)
+
+        self.assertGreater(row_group_size, 1_000_000)
```

Comment:
So I was struggling with how to test this except just empirically with the table that I'm already making.  What I didn't want to do was make a 1Gb temporary file just to test that it has more than one row group, since I thought that would be excessive.  But if you have any more stringent test ideas...